File: Shop-Spa/src/app/_models/Car.ts
import { Photo } from './Photo';

export class Car {
    id: number;
    brand: string;
    model: string;
    engine: string;
    horsePower: number;
    gear: string;
    year: number;
    price: number;
    acceleration: number;
    rent: number;
    endrent: number;
    isrent: boolean;
    photos?: Photo[];
    photoUrl: string;
    userid: number;
    messagerent: string;

}

File: Shop.API/Models/Car.cs
using System;
using System.Collections.Generic;

namespace Shop.API.Models
{
    public class Car
    {
        public int Id { get; set; }
        public string Brand { get; set; }
        public string Model { get; set; }
        public string Engine { get; set; }
        public int HorsePower { get; set; }
        public string Gear { get; set; }
        public int Year { get; set; }
        public int Price { get; set; }
        public int Acceleration { get; set; }
        public DateTime Rent { get; set; }
        public DateTime EndRent { get; set; }
        public ICollection<Photo> Photos { get; set; }

    }
}